src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -110,7 +110,7 @@ abstract protected function describeContainerDefinition(Definition $definition,
 
     abstract protected function describeContainerAlias(Alias $alias, array $options = [], ContainerBuilder $container = null): void;
 
-    abstract protected function describeContainerParameter(mixed $parameter, array $options = []): void;
+    abstract protected function describeContainerParameter(mixed $parameter, ContainerBuilder $containerBuilder, array $options = []): void;
```

Comment:
would it make sense to use $options instead?

Reply:
I thought about it, but the current use of `$options` seems to be conveying the command options.

Reply:
I've pushed https://github.com/symfony/symfony/pull/51011/commits/feb46c6fc4b79ee35a4bfb68389beef9968f9493, WDYT?